frontend/schema/class-schema-article.php

```diff
@@ -52,6 +52,7 @@ public function is_needed() {
 	 */
 	public function generate() {
 		$post          = get_post( $this->context->id );
+		$comments_open = comments_open( $this->context->id );
```

Comment:
I don't want to do this, to be honest, because many sites close their comments after X weeks. Doesn't mean the available comments aren't valid.

Reply:
> 
> 
> I don't want to do this, to be honest, because many sites close their comments after X weeks. Doesn't mean the available comments aren't valid.

True, that's why I considered checking for comments disabled as well as comment count.

Reply:
👍